### DISCUSSION TITLE
TypeError: CTkEntry.__init__() missing 1 required positional argument: 'master'

### DISCUSSION THREAD

**Discussion by mrmorgan-ai** [TomSchimansky/CustomTkinter#1301] [General]
Hi folks, can someone help me with this result? i got this error:TypeError: CTkEntry.init() missing 1 required positional argument: 'master' after i run this line: prompt = ctk.CTkEntry(height=40, width=512,text_font=("Arial",20),text_color="black",fg_color="white")
Any ideas of how i could fix that? pleaase. its been several days

**Comment by its-jd** [comment]
you need to specify the parent of this widget as first argument

`prompt = ctk.CTkEntry(master=self, height=40, width=512,text_font=("Arial",20),text_color="black",fg_color="white")`

if you have defined any other variable as root element for this widget, use that variable instead of self
